services/src/main/java/org/keycloak/social/google/GoogleIdentityProvider.java

```diff
@@ -100,11 +101,11 @@ protected UriBuilder createAuthorizationUrl(AuthenticationRequest request) {
         if (hostedDomain != null) {
             uriBuilder.queryParam(OIDC_PARAMETER_HOSTED_DOMAINS, hostedDomain);
```

Comment:
How does Google behave when setting a comma-separated list of domains?

Reply:
Glad you asked. According to [this docs](https://developers.google.com/identity/openid-connect/openid-connect#hd-param), all `hd` param does is show in the UI that email should ends with that domain.
![image](https://user-images.githubusercontent.com/6848311/216246669-23393c50-ed05-4fe2-a161-a695f73e033d.png)
And off-course, you can enter other email domain like `example@otherdomain.com`. All validation must be done in client side (`keycloak` in this case).

So, in the case of multi-domains, google simply don't show the suggestion as picture above.